nextjs-13/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "nextjs-13",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev -p 3003",
```

Minor preference to change the port at the hundredth unit:

```suggestion
    "dev": "next dev -p 3300",
```

It allows to use sub-ports `3301`, `3302`, etc. for the same application when needed.

---

Woa TIL! [This is the commit with the change](https://github.com/nimblehq/js-framework-benchmark/pull/76/commits/26865c6516980236ce256455a727757d85762092).